source/plugin/data/keys.rst

```diff
@@ -0,0 +1,109 @@
+==========
+Using Keys
+==========
+
+Getting and offering data using a key
+=====================================
+
+Since accessing even a single point of data going from ``DataHolder`` to the correct ``DataManipulator`` tended to
+be quite tedious and bloat both the code and the import list, a more direct way of accessing values via ``Key``\ s
+was devised. Let's just start out with an example.
+
+**Code Example: Healing a data holder, if possible**
+
+.. code-block:: java
+
+    import org.spongepowered.api.data.DataHolder;
+    import org.spongepowered.api.data.key.Keys;
+
+    public void heal(DataHolder target) {
+        if (target.supports(Keys.HEALTH)) {
+            double maxHealth = target.getOrNull(Keys.MAX_HEALTH);
+            target.offer(Keys.HEALTH, maxHealth);
+        }
+    }
+
+Now for the details of the above function.
+
+The first line checks if our given data holder possesses health. Only if he does, he can be healed after all.
+Since a data holder can not have health without having a maximum health and vice versa, a check for one of the
```

Comment:
Basically, since `HEALTH` and `MAX_HEALTH` are treated by the same data processor (as they are part of the same data manipulator), checking if one of them is supported implies the other is too.
Are you saying I should check for both keys, and for the sake of this docs page act like all keys are independent from each other?